src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -70,7 +70,8 @@ public interface IBlockRepository : IDisposable
         Task<uint256> GetTrxBlockIdAsync(uint256 trxid);
 
         /// <summary>
-        /// For every block in the database, store the transactions (or don't) according to whether TxIndex is set.
+        /// Iterate over every block in the database. 
+        /// If <see cref="TxIndex"/> is true, we store the block hash alongside the transaction hash in the transaction table, otherwise clear the transaction table.
```

Comment:
`<c>true</c>`

Reply:
however, there is no TxIndx on the interface, so it is weird to have it in the comment

Reply:
ah sorry, there is TxIndex